e2e/db.js

```diff
@@ -0,0 +1,39 @@
+import mongoose, { mongo } from "mongoose";
+import { MongoMemoryServer } from "mongodb-memory-server";
+
+let mongoServer = null;
+let dbInstance = undefined;
+let MONGO_PORT = process.env.MONGODB_PORT || 10000;
+let MONGO_URL = `mongodb://127.0.0.1:${MONGO_PORT}/`;
```

Comment:
refactor this to take direct mongodb url from env

Values that are not re-assigned later should be marked as `const` please double check

Reply:
Naming convention mismatch, at `.env` level you are calling them as `MONGODB_xxx`
But at variable level it became `MONGO_xxx`

Make should all of them are in `MONGODB_xxx` format only